modules/library/coverage/src/main/java/org/geotools/coverage/grid/RenderedSampleDimension.java

```diff
@@ -142,15 +139,11 @@ static boolean create(final CharSequence        name,
             }
             sd = new RenderedSampleDimension(sd, image, i);
             dst[i] = sd;
-            if (sd.geophysics(true ) == sd) countGeophysics++;
-            if (sd.geophysics(false) == sd) countIndexed++;
+            countGeophysics++;
```

Comment:
Now that we don't have a concept of geophisics it would be better to rename these variables to something else.
There is still a lot of references in the code:
````
> find . -name "*.java" | xargs grep geophysics | wc -l
120
````